app/config/doctrine.yml

```diff
@@ -37,3 +44,9 @@ services:
         class: Common\Doctrine\Entity\CreateSchema
         arguments:
             - "@doctrine.orm.entity_manager"
+
+    repository.meta:
```

Comment:
Just thinking out loud here:
* 'common.repository.meta', 'common.repository.module_extra', ...?
* Or 'fork.repository.meta', 'fork.repository.module_extra' (to be consistent with 'fork.settings', ...)